collectors/okta/okta_collector.js

```diff
@@ -76,8 +77,18 @@ class OktaCollector extends PawsCollector {
         .catch((error) => {
             error.errorCode = this._isNoErrorCode(error);
             if (this._isThrottlingError(error)) {
-                collector.reportApiThrottling(function() {
-                    return callback(error);
+                // if x-rate-limit-reset value return by api then accordingly delay the api call to avoid throttle error again other wise increase the delay by 1min till max 15min.
```

Comment:
could you pls create a new function and add this logic there.

Reply:
move the code in _isThrottlingError